app/models/tag.rb

```diff
@@ -239,7 +239,6 @@ def self.trending(limit = 5 , start_date = DateTime.now - 1.month , end_date = D
   #select nodes by tagname and user_id
   def self.tagged_nodes_by_author(tagname, user_id)
     if tagname[-1..-1] == '*'
-      @wildcard = true
```

Comment:
Ah, we use this to disable some things that don't currently work with wildcard searches... so maybe preserve this unless we're sure that the whole page works identically for normal tags and wildcard queries?